core/server/master/src/main/java/alluxio/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -70,13 +74,17 @@ public FileSystemMasterClientServiceHandler(FileSystemMaster fileSystemMaster) {
 
   @Override
   public long getServiceVersion() {
-    return Constants.FILE_SYSTEM_MASTER_CLIENT_SERVICE_VERSION;
+    LOG.debug("Enter GetServiceVersion.");
+    long ret = Constants.FILE_SYSTEM_MASTER_CLIENT_SERVICE_VERSION;
+    LOG.debug("Exit GetServiceVersion.");
+    return ret;
```

Comment:
I think this may be unnecessary since like `sessionHeartbeat`, this can cause a lot of noise.